### PR TITLE
Add temporal info on pt route

### PR DESCRIPTION
**Pull request by nlehuby**
Add support for the following tags on `Route` and `Line`, that are standard osm tags since the [proposal approved in 2018](https://wiki.openstreetmap.org/wiki/Proposed_features/Public_transport_schedules/Interval):
* `interval`
* `opening_hours`
* `interval:conditional` 
* `duration` (`Route` only)

We may need to talk about how we want to name them in osm-tc-extractor :thinking:

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -103,6 +103,10 @@ pub struct Route {
     pub operator: String,
     pub network: String,
     pub mode: String,
+    pub interval: String,
```
Comment (nlehuby) [inline src/lib.rs]: interval ? headway ? frequency ?

File: src/lib.rs [hunk 2]
```diff
@@ -103,6 +103,10 @@ pub struct Route {
...
+    pub opening_hours: String,
+    pub interval_conditional: String,
```
Comment (nlehuby) [inline src/lib.rs]: interval_exceptions ?

File: src/lib.rs [hunk 3]
```diff
@@ -103,6 +103,10 @@ pub struct Route {
...
+    pub duration: String,
```
Comment (nlehuby) [inline src/lib.rs]: travel_time ?

File: src/lib.rs (hunk 1 above)
Comment (prhod) [reply src/lib.rs]: IHMO `interval` is good if you know what this is about. I'm not fan of `headway`. Is rename it to `frequency` (even if it's a number of minutes) seems a good idea ?

File: src/lib.rs (hunk 3 above)
Comment (prhod) [reply src/lib.rs]: +1 for `travel_time` ^^

File: src/lib.rs (hunk 2 above)
Comment (prhod) [reply src/lib.rs]: `exception` seems better than `conditional` to me.

**Comment by prhod** [comment]
Thanks for adding those tags to the standard extraction :+1:

File: src/lib.rs (hunk 1 above)
Comment (nlehuby) [reply src/lib.rs]: `frequency` seems ok in the public transport context I think

**Comment by nlehuby** [comment]
Thanks :+1: can we have a release with this new feature please ?

**Comment by prhod** [comment]
done, with the ready to use binary :wink:

**Pull request merged**